src/components/Timereport.js

```diff
@@ -70,9 +75,11 @@ class Timereport extends Component {
     }
 
     fetchData = async (selectedUserId, startDate, endDate, selectedMonth) => {
-        const [data, lockstate] = await Promise.all([
+        const [data, lockstate, totaldays] = await Promise.all([
             fetch(`${this.props.backend_url}/users/${selectedUserId}/events?from=${encodeURIComponent(startDate)}&to=${encodeURIComponent(endDate)}`).then(response => response.json()),
             fetch(`${this.props.backend_url}/locks/dates/${selectedMonth}`).then(response => response.json()),
+
+            fetch(`https://api2.codelabs.se/${encodeURIComponent(selectedMonth)}.json`).then(response=>response.json()),
```

Comment:
Very small detail, but it seem this deviates from the other standard (no spaces in `=>`).
Maybe we should have auto linting in pipeline?